packages/interactivity/src/router.js

```diff
@@ -55,8 +55,8 @@ const regionsToVdom = ( dom ) => {
 		const id = region.getAttribute( attrName );
 		regions[ id ] = toVdom( region );
 	} );
-
-	return { regions };
+	const title = dom.querySelector( 'title' ).innerText || null;
```

Comment:
I think it's safer to use the optional chaining operator here. Note that `querySelector()` could return `null` in the hypothetical case `<title>` is missing, thus throwing an error.

Regarding the logical OR operator at the end, we can get rid of it, as the value of `title` would be falsy anyway.

```suggestion
	const title = dom.querySelector( 'title' )?.innerText;
```